packages/dataviews/src/components/dataform-field-visibility/index.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { NormalizedField } from '../../types';
+
+type DataFormFieldVisibilityProps< Item > = React.PropsWithChildren< {
+	field: NormalizedField< Item >;
+	data: Item;
+} >;
+
+export default function DataFormFieldVisibility< Item >( {
```

Comment:
We could remove the `Data` prefix to this component and simply be`FormFieldVisibility`.

Reply:
Renamed as part of this commit: https://github.com/WordPress/gutenberg/pull/65826/commits/3f73afb6da4a850088ada24bbbb21e85665935e5